### PR TITLE
add mempool options to advanced

### PR DESCRIPTION
**Pull request by MattDHill**

### CODE REVIEW

File: assets/bitcoin.conf.template [hunk 1]
```diff
@@ -22,6 +22,16 @@ rpcservertimeout={{rpc.advanced.servertimeout}}
 rpcthreads={{rpc.advanced.threads}}
 rpcworkqueue={{rpc.advanced.workqueue}}
 
+## MEMPOOL
+{{#IF advanced.mempool.persistmempool
+persistmempool=1
+}}
+{{IF !persistmempool
```
Comment (dr-bonez) [inline assets/bitcoin.conf.template:29]:
```suggestion
{{IF !advanced.mempool.persistmempool
```

File: instructions.md [hunk 2]
```diff
@@ -1 +0,0 @@
-docs/instructions.md
```
Comment (dr-bonez) [inline instructions.md:1]: Is there a reason you deleted this? it is needed to have appmgr pack the instructions into the s9pk.

**Review by dr-bonez — CHANGES_REQUESTED** [review]

File: instructions.md (hunk 2 above)
Comment (MattDHill) [reply instructions.md:1]: Nope, thought it was left over

File: assets/bitcoin.conf.template (hunk 1 above)
Comment (MattDHill) [reply assets/bitcoin.conf.template:29]: I was following the convention from the listing beneath this one

Comment (dr-bonez) [reply assets/bitcoin.conf.template:29]: the listing beneath this one has a bug :(

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**